lib/cli.rb

```diff
@@ -1,11 +1,15 @@
 # frozen_string_literal: true
 
-# Resonsible for any input/output via the command line
+# Responsible for any input/output via the command line
 class CLI
   def display_message(text)
     puts text
   end
 
+  def prompt_user_input
+    gets.chomp
+  end
+
   def display_board(board)
     formatted_board = format_board_for_display board
```

Comment:
I didn't get to address this in my last PR review but you stated that you wanted board display logic in the CLI class because a board string is specific to the terminal. What would happen if I rendered this string in the browser, for example in a <p> tag?

Reply:
I think I could move the logic from `CLI#format_board_for_display` to a new method in `Board` called `to_s`. `CLI#format_board_for_display` could call `board.to_s`. Do you think that would make sense?

Alternatively, I could create a class/utility with methods that would format the board. But, I think this would be overkill at this point considering it would have one method that's in one place.

Reply:
I think it's less important to think about classes that are too small or too large and think about the role that the class should fulfill. The question I would ask is - In the domain of the game Tic Tac Toe, does a board know how to turn itself into a string? Is that logic appropriate in this domain? It can be powerful to think of OOP and your objects in terms of abstractions of the domain you are trying to model.

Reply:
I think I was a little confused about this one. Maybe we could talk about it during tomorrows IPM.

I think it would be okay for CLI to handle converting the board to a string. We may need to use it in different places, such as printing it in the browser. But, I feel like we could break it out of CLI when we get there.